test/unit/processing/statement/get-statement.test.js

```diff
@@ -28,14 +34,16 @@ describe('get various components and transform to statement object', () => {
     const retrievedCalculationData = JSON.parse(JSON.stringify(require('../../../mock-objects/mock-calculation').rawCalculationData))
     const retrievedFundingsData = JSON.parse(JSON.stringify(require('../../../mock-objects/mock-fundings').rawFundingsData))
     const retrievedPaymentRequest = JSON.parse(JSON.stringify(require('../../../mock-payment-request').processingPaymentRequest))
-    const retreivedSettlement = JSON.parse(JSON.stringify(require('../../../mock-settlement')))
+    const retrievedSettlement = JSON.parse(JSON.stringify(require('../../../mock-settlement')))
+    const retrievedLatestPayment = JSON.parse(JSON.stringify(require('../../../mock-objects/mock-latest-payment')))
 
     settlement = {
-      invoiceNumber: retreivedSettlement.invoiceNumber,
+      invoiceNumber: retrievedSettlement.invoiceNumber,
       paymentRequestId: 1,
-      reference: retreivedSettlement.reference,
-      settled: retreivedSettlement.settled,
-      settlementDate: new Date(retreivedSettlement.settlementDate)
+      reference: retrievedSettlement.reference,
+      settled: retrievedSettlement.settled,
+      settlementDate: new Date(retrievedSettlement.settlementDate),
+      value: retrievedSettlement.value
 

```

Comment:
Whitespace of doom -- `standard` not pick this up?

Reply:
It was a rogue new line rather than whitespace.  Standard thinks one is ok, but draws the line at two.